### DISCUSSION TITLE
[New Feature] Change SelfHeal from a global to a unit setting

### DISCUSSION THREAD

**Discussion by askhati** [Vinifera-Developers/Vinifera#1530] [Feature Requests]
### Description:
Instead of having the SelfHeal logic defined on a global level (to determine the number of HP healed per game time increment), move this down to a unit level and keep the global setting only as a default.

### Possible Implementation:
Create new tag: "SelfHealPoints=N" (N is an integer >0)
Unit with this tag will heal N hit points per global game time increment.

Optionally:
Create secondary tag: "SelfHealROF=M (M is an integer >0)
If present on the unit, the SelfHealPoints=N amount will be applied every M time increments; if not present, default to the global time increment setting.

**Comment by JoyfulShush** [comment]
Great QoL for customizability. Somewhat similar to discussion #1450
